models/model.js

```diff
@@ -68,4 +68,14 @@ exports.selectUsers=()=>{
     return db.query("SELECT * FROM users;").then((result)=>{
         return result.rows
     })
+}
+
+exports.deleteCommentByID=(id)=>{
+    return db.query('SELECT * FROM comments WHERE comment_id=$1', [id]).then((result)=>{
```

Comment:
Rarther than using an extra database call to check the comment exists before deleting you can do this in one call. I'd have a look what the delete query returns depending on if the ID exists or doesn't